cmd/ksync/commands/blocksync.go

```diff
@@ -58,9 +55,14 @@ var blockSyncCmd = &cobra.Command{
 		chainRest = utils.GetChainRest(chainId, chainRest)
 		storageRest = strings.TrimSuffix(storageRest, "/")
 
+		// if no home path was given get the default one
+		if homePath == "" {
+			homePath = utils.GetHomePathFromBinary(binaryPath)
+		}
+
 		backupCfg, err := backup.GetBackupConfig(homePath, backupInterval, backupKeepRecent, backupCompression, backupDest)
 		if err != nil {
-			logger.Error().Str("err", err.Error()).Msg("failed to create backup config")
+			logger.Error().Msg(err.Error())
```

Comment:
Please change to logger.Error().Str("err", err.Error()).Msg("could not get backup config")

Reply:
done